snapcraft/internal/sources/__init__.py

```diff
@@ -111,7 +111,12 @@
         "snap": Snap,
         "": Local,
     }
+elif sys.platform == "win32":
+    from ._git import Git  # noqa
+    from ._local import Local  # noqa
+    from ._tar import Tar  # noqa
```

Comment:
Tar is missing from the list in the commit message

Reply:
Fixed!